src/controllers/ChargeController.php

```diff
@@ -64,7 +64,10 @@ public function actions()
                 'on beforePerform' => function (Event $event) {
                     /** @var ChargeQuery $query */
                     $query = $event->sender->getDataProvider()->query;
-                    $query->withCommonObject()->withLatestCommonObject();
+                    $query
+                      ->withCommonObject()
+                      ->withLatestCommonObject()
+                      ->andWhere(['with_rootFullType' => true]);
```

Comment:
```suggestion
                      ->withRootChargeType();
```

And implement this method in `ChargeQuery`, as done for `withCommonObject()`